src/WinUIEx/MonitorInfo.cs

```diff
@@ -32,10 +33,34 @@ public unsafe static IList<MonitorInfo> GetDisplayMonitors()
             return list;
         }
 
+        /// <summary>
+        /// Gets the display monitor that is nearest to a given window.
+        /// </summary>
+        /// <param name="hwnd">Window handle</param>
+        /// <returns>The display monitor that is nearest to a given window.</returns>
+        public unsafe static MonitorInfo GetWindowDisplayMonitor(IntPtr hwnd)
```

Comment:
Probably better to just name this `MonitorFromWindow` to match the interop method. What's the difference between a "monitor" and a "display monitor" ?